python-checks/src/test/resources/checks/numpyWhereOneCondition.py

```diff
@@ -0,0 +1,47 @@
+def import_1():
+    import numpy as np
+    arr = np.array([1, 2, 3, 4])
+
+    empty = np.where()
+    args = (arr > 1, arr + 1, arr)
+    result = np.where(*args)
+    indices = np.nonzero(arr > 1)
+
+    result = np.where(arr > 2)  # Noncompliant {{Use "np.nonzero" when only the condition parameter is provided to "np.where".}}
+    #        ^^^^^^^^^^^^^^^^^
+
+    result = np.where(arr > 1, arr + 1, arr)
+    indices = np.nonzero(arr > 1)
+
+    result = xx.where(arr > 2)
+
+    result = np.where(arr > 1, arr + 1)  # Compliant: There are two arguments, so we don't correct the rule. This however is an error.
+    indices = np.nonzero(arr > 1)
+
+
+def import_2():
+    import numpy as xx
+    arr = xx.array([1, 2, 3, 4])
+
+    result = xx.where(arr > 2)  # Noncompliant
+    #        ^^^^^^^^^^^^^^^^^
+
+    result = xx.where(arr > 1, arr + 1, arr)
+    indices = xx.nonzero(arr > 1)
+
+    result = np.where(arr > 2)
```

Comment:
One missing test could be when the user provides the parameter keyword as well. I don't think we want to raise the issue if the user does weird stuff like `np.where(x=arr+1)` it will be confusing to raise the issue with the message about using `np.nonzero` because it does not seem like it is the intention. Feel free to challenge here as I am not 100% sure it is worth the effort.